GOVERNANCE.md

```diff
@@ -0,0 +1,44 @@
+NOTE: The following is a snapshot of a perpetually changing state of affairs in
+the WGPU community. It is not a binding contract between users and
+decision-makers of the WGPU project.
+
+---
+
+The **WGPU project** is an ecosystem that _enables application authors to write
+portable and performant graphics programs_. It was originally conceived to
+provide an implementation of WebGPU for Firefox as the standard evolved and
+settled into something that could be shipped on all web browsers. WGPU has also
+enjoyed much contribution and use from other projects that requires graphics
+programming. We expect that these sorts of users will continue for the lifetime
+of project, and we embrace these contributors' needs and effort as the lifeblood
+of WGPU.
+
+## Mission
+
+The WGPU community seeks to realize the following directives through the
+project: it…
+
+1. …provides an implementation of the WebGPU API that is…
+    1. …correct and fully conformant.
+    1. …portable across all major platforms.
+    1. …performant enough to enable demanding applications.
+1. …serves as a platform of experimentation for:
+    1. …WebGPU standards development.
+    1. …native application authors.
+
+## Decision-making
+
+The WGPU community's decision-making is influenced by the following
+groups:
+
+* Connor Fitzgerald (@cwfitzgerald)
+* Firefox's WebGPU team
+* Other users that ship applications based on WGPU
```

Comment:
We should mention deno and crowlkats

Reply:
I was careful to avoid mentioning groups smaller than Firefox (because I don't want details that are prone to changing in a document that seems frankly unlikely to get updated often), but if you'd like them explicitly mentioned, then SGTM!

Reply:
We have an explicitly special relationship with deno (we host deno_webgpu!) which has existed for a few years, I don't expect this to really change. We could also put him under the community banner, as I think of him as part of community interest.

Reply:
Noted Deno and @crowlKats with a62303575.

I've added Firefox contributor names with a62303575 (minus @bradwerth, unless he also wants to be on there), though I think maybe that's not a good idea for us, since membership and contribution can fluctuate over time? CC @jimblandy, @teoxoy.

Reply:
Resolving for now, but feel free to un-resolve if somebody has an objection to the current contents.